unified-installer/astra-unified-installer.sh

```diff
@@ -188,6 +209,90 @@ get_configs() {
+      - Trident ACP:         private.registry.io/some/repository/path/trident-acp:24.02
+      - Connector Operator:  private.registry.io/some/repository/path/astra-connector-operator:202405211614-main
+
+    An appropriate configuration might be:
+      - IMAGE_REGISTRY="private.registry.io"
+      - IMAGE_BASE_REPO="some/repository/path"
+      - TRIDENT_IMAGE_TAG="24.02"
+      - CONNECTOR_OPERATOR_IMAGE_TAG="202405211614-main"
+
+  IMAGE_REGISTRY                     The default Docker registry to pull all images from. Overridden by values below.
+  TRIDENT_OPERATOR_IMAGE_REGISTRY    The Docker registry to pull the Trident Operator image from.
+  TRIDENT_AUTOSUPPORT_IMAGE_REGISTRY The Docker registry to pull the Trident Autosupport image from.
+  TRIDENT_IMAGE_REGISTRY             The Docker registry to pull the Trident image from.
+  CONNECTOR_OPERATOR_IMAGE_REGISTRY  The Docker registry to pull the Connector Operator image from.
+  CONNECTOR_IMAGE_REGISTRY           The Docker registry to pull the Connector image from.
+  NEPTUNE_IMAGE_REGISTRY             The Docker registry to pull the Neptune image from.
+  TRIDENT_ACP_IMAGE_REGISTRY         The Docker registry to pull the Trident ACP image from.
+
+  IMAGE_BASE_REPO                    The default base repository path (i.e. excluding the image name) for all images. Overridden by values below.
+  TRIDENT_OPERATOR_IMAGE_REPO        The repository path for the Trident Operator image.
+  TRIDENT_AUTOSUPPORT_IMAGE_REPO     The repository path for the Trident Autosupport image.
+  TRIDENT_IMAGE_REPO                 The repository path for the Trident image.
+  CONNECTOR_OPERATOR_IMAGE_REPO      The repository path for the Connector Operator image.
+  CONNECTOR_IMAGE_REPO               The repository path for the Connector image.
+  NEPTUNE_IMAGE_REPO                 The repository path for the Neptune image.
+  TRIDENT_ACP_IMAGE_REPO             The repository path for the Trident ACP image.
+
+  TRIDENT_IMAGE_TAG                  The tag for the Trident image. Overridden by other 'TRIDENT_' values below.
+  TRIDENT_OPERATOR_IMAGE_TAG         The tag for the Trident Operator image.
+  TRIDENT_AUTOSUPPORT_IMAGE_TAG      The tag for the Trident Autosupport image.
+  TRIDENT_ACP_IMAGE_TAG              The tag for the Trident ACP image.
+  CONNECTOR_OPERATOR_IMAGE_TAG       The tag for the Connector Operator image.
+
+  ----- Connector Configuration
+  *ASTRA_CONTROL_URL                 The Astra Control URL. Required, provided by the Astra Control UI.
+  *ASTRA_API_TOKEN                   The Astra API token. Required, provided by the Astra Control UI.
+  *ASTRA_ACCOUNT_ID                  The Astra account ID. Required, provided by the Astra Control UI.
+  *ASTRA_CLOUD_ID                    The Astra cloud ID. Required, provided by the Astra Control UI.
+  *ASTRA_CLUSTER_ID                  The Astra cluster ID. Required, provided by the Astra Control UI.
```

Comment:
I thought we weren’t making these required after all?

Reply:
You're right, I forgot we were doing that. I'll change the message.

Reply:
After looking into this, the changes were made on the astra-connector side, but we still need to tweak the script to support optional CLOUD_ID/CLUSTER_ID. I created a 1pt card to take care of this: https://jira.ngage.netapp.com/browse/ASTRACTL-33865